core/chains/evm/txmgr/eth_broadcaster.go

```diff
@@ -375,47 +375,46 @@ func (eb *EthBroadcaster) handleInProgressEthTx(ctx context.Context, etx EthTx,
 		return errors.Wrap(err, "building transmit checker")
 	}
 
+	lgr, err := etx.GetLogger(eb.logger.With(
+		"gasPrice", attempt.GasPrice,
+		"gasTipCap", attempt.GasTipCap,
+		"gasFeeCap", attempt.GasFeeCap,
+	))
+	if err != nil {
+		eb.logger.Errorw("failed to get tx logger meta of the transaction", "err", err)
```

Comment:
qq: what happens here for the rest of the func call if logger errors and lggr is nil? not sure but probably need to fallback if eth meta is missing

Reply:
Fixed